### PR TITLE
New package: hfsutils

### PR DESCRIPTION
**Pull request by stenstorp**
Required for manipulating Apple Partition Map bootstrap partitions.

### CODE REVIEW

File: srcpkgs/hfsutils/template [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+build_style=gnu-configure
+short_desc="A comprehensive software package to permit manipulation of HFS volumes"
+maintainer="Thomas Batten <stenstorpmc@gmail.com>"
+license="GPL-2"
+homepage="https://www.mars.org/home/rob/proj/hfs/"
+distfiles="ftp://ftp.mars.org/pub/hfs/hfsutils-${version}.tar.gz"
+checksum="bc9d22d6d252b920ec9cdf18e00b7655a6189b3f34f42e58d5bb152957289840"
+
+do_install() {
+  vbin hattrib
```
Comment (maxice8) [inline srcpkgs/hfsutils/template]: wrong indent

File: srcpkgs/hfsutils/template [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Template file for hfsutils
+
+pkgname="hfsutils"
+version="3.2.6"
+revision=1
+build_style=gnu-configure
+short_desc="A comprehensive software package to permit manipulation of HFS volumes"
+maintainer="Thomas Batten <stenstorpmc@gmail.com>"
+license="GPL-2"
```
Comment (maxice8) [inline srcpkgs/hfsutils/template]: use spdx identifier

File: srcpkgs/hfsutils/template [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+homepage="https://www.mars.org/home/rob/proj/hfs/"
+distfiles="ftp://ftp.mars.org/pub/hfs/hfsutils-${version}.tar.gz"
```
Comment (maxice8) [inline srcpkgs/hfsutils/template]: do not use ftp if posslbe

File: srcpkgs/hfsutils/template [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Template file for hfsutils
+
```
Comment (maxice8) [inline srcpkgs/hfsutils/template]: remove

File: srcpkgs/hfsutils/template [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+pkgname="hfsutils"
+version="3.2.6"
```
Comment (maxice8) [inline srcpkgs/hfsutils/template]: unquote

File: srcpkgs/hfsutils/template [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+  vbin hls
+  vbin hmkdir
+  vbin hmount
+  vbin hpwd
+  vbin hrename
+  vbin hrmdir
+  vbin humount
+  vbin hvol
+
+  vman doc/man/hattrib.1
```
Comment (maxice8) [inline srcpkgs/hfsutils/template]: if possible do in for-loop

File: srcpkgs/hfsutils/template [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+pkgname="hfsutils"
+version="3.2.6"
+revision=1
+build_style=gnu-configure
+short_desc="A comprehensive software package to permit manipulation of HFS volumes"
+maintainer="Thomas Batten <stenstorpmc@gmail.com>"
+license="GPL-2"
+homepage="https://www.mars.org/home/rob/proj/hfs/"
+distfiles="ftp://ftp.mars.org/pub/hfs/hfsutils-${version}.tar.gz"
+checksum="bc9d22d6d252b920ec9cdf18e00b7655a6189b3f34f42e58d5bb152957289840"
```
Comment (maxice8) [inline srcpkgs/hfsutils/template]: unquote

File: srcpkgs/hfsutils/template [hunk 8]
```diff
@@ -0,0 +1,47 @@
+# Template file for hfsutils
```
Comment (maxice8) [inline srcpkgs/hfsutils/template]: header is wrong

**Comment by maxice8** [comment]
Use `xlint` / fix erorrs pointed out by xlint

File: srcpkgs/hfsutils/template (hunk 3 above)
Comment (stenstorp) [reply srcpkgs/hfsutils/template]: hfsutils is only officially available via ftp unless you want the debian re-package of it.

**Comment by maxice8** [comment]
On Tue, Jan 29, 2019 at 12:38:33PM -0800, stenstorp wrote:
> stenstorp commented on this pull request.
> 
> 
> 
> > @@ -0,0 +1,47 @@
> +# Template file for hfsutils
> +
> +pkgname="hfsutils"
> +version="3.2.6"
> +revision=1
> +build_style=gnu-configure
> +short_desc="A comprehensive software package to permit manipulation of HFS volumes"
> +maintainer="Thomas Batten <stenstorpmc@gmail.com>"
> +license="GPL-2"
> +homepage="https://www.mars.org/home/rob/proj/hfs/"
> +distfiles="ftp://ftp.mars.org/pub/hfs/hfsutils-${version}.tar.gz"
> 
> hfsutils is only officially available via ftp unless you want the debian re-package of it.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/7940#discussion_r252007268

bummer, ftp is really bad against Travis

**Comment by maxice8** [comment]
Please check for cross-compilation

**Comment by stenstorp** [comment]
Package cross compiles successfully for all supported arcitectures

**Pull request merged**